components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/ActionManagementServiceImpl.java

```diff
@@ -48,6 +49,7 @@ public class ActionManagementServiceImpl implements ActionManagementService {
     private static final CacheBackedActionMgtDAO CACHE_BACKED_DAO =
             new CacheBackedActionMgtDAO(new ActionManagementDAOImpl());
     private static final ActionValidator ACTION_VALIDATOR = new ActionValidator();
+    private static final ActionMgtAuditLoggerUtil auditLogger = new ActionMgtAuditLoggerUtil();
```

Comment:
Shall we say ActionManagementAuditLogger. So it's more consistent with naming used now

Reply:
This is updated with b5355995d46dbafa80bb825fa0da60e5bd5062ee

Reply:
This is updated with b5355995d46dbafa80bb825fa0da60e5bd5062ee